versions.props

```diff
@@ -27,6 +27,7 @@ com.google.cloud:libraries-bom = 24.1.0
 org.scala-lang.modules:scala-collection-compat_2.12 = 2.6.0
 org.scala-lang.modules:scala-collection-compat_2.13 = 2.6.0
 com.emc.ecs:object-client-bundle = 3.3.2
+org.immutables:value = 2.9.0
```

Comment:
It looks like this pulls in another copy of Guava? https://search.maven.org/artifact/org.immutables/value/2.9.2/jar

Reply:
all dependencies that `immutables.value` requires are properly repackaged and the guava classes live under `org.immutables.value.internal.$guava$`. However, I didn't realize that they just released a new version, so I'm upgrading this to `2.9.2`